tests/kickstarts/test_suite.cfg

```diff
@@ -41,6 +41,11 @@ rootpw --iscrypted $6$rhel6usgcb$aS6oPGXcPKp3OtFArSrhRwu6sN8q2.yEGY7AIwDOQd23YCt
 # Plaintext password is: admin123
 user --name=admin --groups=wheel --password=$6$Ga6ZnIlytrWpuCzO$q0LqT1USHpahzUafQM9jyHCY9BiE5/ahXLNWUMiVQnFGblu0WWGZ1e6icTaCGO4GNgZNtspp1Let/qpM7FMVB0 --iscrypted
 
+# Configure firewall settings for the system (optional)
+# --enabled	reject incoming connections that are not in response to outbound requests
+# --ssh		allow sshd service through the firewall
+firewall --enabled --ssh
```

Comment:
Why enable firewall?

Reply:
I would say that usually you want your machine installed with firewall enabled by default. What is your reasoning to not enable it?

Reply:
I think `firewalld` is enabled by default and ssh service is enabled also.